demo/settings.py

```diff
@@ -124,4 +124,15 @@
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
 
-MEILISEARCH = {}
+MEILISEARCH = {
+    'HTTPS': False,  # Whether HTTPS is enabled for the meilisearch server
+    'HOST': 'localhost',  # The host for the meilisearch server
+    'MASTER_KEY': "9D3CCB9680E1EB7040490171C9F8D4E7",  # The master key for meilisearch. See https://www.meilisearch.com/docs/learn/security/basic_security for more detail
+    'PORT': 7700,  # The port for the meilisearch server
+    'TIMEOUT': None,  # The timeout to wait for when using sync meilisearch server
```

Comment:
not sure about this.. but was getting errors about auth key.. and didnt see mocks for it.. so i guess you have some local setup?

also checked github workflows for setup but didnt see anythign remarkable... 

but i guess you run meili local with no master key?